mlflow/java/client/src/test/java/org/mlflow/tracking/MlflowClientTest.java

```diff
@@ -171,6 +171,39 @@ public void addGetRun() {
             tag -> tag.getKey().equals("mlflow.parentRunId")));
   }
 
+  @Test
+  public void deleteTag() {
+    // Create experiment
+    String expName = createExperimentName();
+    String expId = client.createExperiment(expName);
+
+    // Create run
+    RunInfo runCreated = client.createRun(expId);
+    String runId = runCreated.getRunUuid();
+    client.setTag(runId, "tag0", "val0");
+    client.setTag(runId, "tag1", "val1");
+    client.deleteTag(runId, "tag0");
+    Run run = client.getRun(runId);
+    // test that the tag was correctly deleted.
+    for (RunTag rt : run.getData().getTagsList()) {
+      Assert.assertTrue(!rt.getKey().equals("tag0"));
+    }
+    // test that you can't delete a tag that doesn't already exist.
+    try {
+      client.deleteTag(runId, "fakeTag");
```

Comment:
would be interesting to test `deleteTag` on `"tag0"` (again) after confirming that it was deleted.

Reply:
Added such a test.